src/components/CreatorPage/Main/MainComponents/Table.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import "./Table.scss";
+import data from "../../CreatorContent/Data/data";
+
+const Table = () => {
+  const headings = ["Videos", "Music", "Pictures"];
+  const [selected, setSelected] = useState([true, false, false]);
+
+  return (
+    <div className="Table">
+      <>
+        <ul className="Table__headings">
+          {headings.map((heading, index) => {
+            return (
+              <li
+                className={selected[index] ? "Table__headings-underline" : ""}
+                key={index}
+                onClick={() => {
+                  const newSelected = [false, false, false];
+                  newSelected[index] = true;
+                  setSelected(newSelected);
+                }}
+              >
+                <Link to="">{heading}</Link>
```

Comment:
On clicking the Nav items of Content Navigation Bar, we shouldn't redirect user to a new route using Link. Instead, we just replace the data inside the data variable which you have called map() function on.

Reply:
Yes sir that will do the same thing... This will be done using nested routing, the same way as clicking on the links in the side menu just changes the content in the specific area...
Only the content inside the table will change not the entire page...

Reply:
Okay cool!